Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM node:14-alpine
-WORKDIR /usr/src/app
-COPY package.json ./
-RUN npm install
-COPY ./src ./src
-CMD npm start
\ No newline at end of file
+FROM rust:1.68.1 AS builder
+RUN apt-get update && apt-get -y upgrade
+COPY . .
+RUN cargo build --release
+
+FROM debian:buster-slim
+COPY --from=builder ./target/release/blob-indexer ./target/release/blob-indexer
```

Comment:
I have tried this dockerfile and the executable fails because it cannot find `libssl.so.1.1`. You should install libssl1.1 from the second imagen:

`RUN apt-get update && apt-get -y install libssl1.1`